tests/plugins/balloontoolbar/refreshposition.js

```diff
@@ -0,0 +1,88 @@
+/* bender-tags: balloontoolbar */
+/* bender-ckeditor-plugins: toolbar,balloontoolbar,floatingspace,image2,undo */
+/* bender-include: _helpers/default.js */
+/* global ignoreUnsupportedEnvironment */
+
+( function() {
+	'use strict';
+
+	var commonConfig = {
+		allowedContent: true,
+		height: 200
+	};
+
+	bender.editors = {
+		classic: {
+			config: commonConfig
+		},
+
+		divarea: {
+			config: CKEDITOR.tools.object.merge( commonConfig, { extraPlugins: 'divarea' } )
+		},
+
+		inline: {
+			creator: 'inline',
+			config: commonConfig
+		}
+	};
+
+	var parentFrame = window.frameElement,
+		originalHeight = parentFrame && parentFrame.style.height;
+
+	var tests = {
+		setUp: function() {
+			// In IE8 tests are run in very small window which breaks positioning assertions and tests fails (#1076).
+			if ( CKEDITOR.env.ie && CKEDITOR.env.version < 9 ) {
```

Comment:
This part is done by `ignoreUnsupportedEnvironment` function that you call at the end of a file.